docs/making-your-own.md

```diff
@@ -135,31 +123,44 @@ see [Local Testing](/local-testing)
 - Installing SDDM the way it is doesn't work as the SDDM user needs to exist: [github issue](https://github.com/ublue-os/main/issues/224)
     - You can get around this by executing `adduser sddm` before running sddm
 
-# Best Practices
+## Best Practices
 
 The following are some best practices based on experience with learning how to move to this new model:
 
+### What repo to fork?
```

Comment:
Moving it down here works as well, as forking something like 'main' is still is an option if that's what people want